.github/workflows/publish-npm.yml

```diff
@@ -24,17 +26,19 @@ jobs:
 
       - run: pnpm install
 
-      - run: cd packages/mantle && pnpm publish --access public --no-git-checks
+      - name: Create Release Pull Request or Publish to npm
+        uses: changesets/action@v1
+        with:
+          publish: pnpm release
+          version: pnpm version
         env:
-          NODE_AUTH_TOKEN: ${{ secrets.NPM_PUBLISH_TOKEN }}
-
-      - name: Grab the released version and store it in an env variable
-        run: |
-          VERSION=$(jq -r '.version' ./packages/mantle/package.json)
-          echo "MANTLE_VERSION=$VERSION" >> $GITHUB_ENV
```

Comment:
We get outputs from the changesets action, so we can remove this step 🙂 https://github.com/changesets/action?tab=readme-ov-file#outputs